src/chrome/content/rules/BME.hu_incomplete.xml

```diff
@@ -27,10 +27,6 @@
 	<target host="cs.bme.hu" />
 	<target host="www.cs.bme.hu" />
 
-	<!-- Workaround https://mit.bme.hu cert error -->
```

Comment:
Can you also remove the `<!-- !!! -->` next to the target? It's hard to understand why it was put there now and it might discourage other contributors to edit this file because they could feel like there is a hidden issue they don't understand.